code/modules/mob/living/carbon/xenomorph/castes/crusher/abilities_crusher.dm

```diff
@@ -66,8 +63,8 @@
 	action_icon_state = "cresttoss"
 	mechanics_text = "Fling an adjacent target over and behind you. Also works over barricades."
 	ability_name = "crest toss"
-	plasma_cost = 100
-	cooldown_timer = 18 SECONDS
+	plasma_cost = 60
+	cooldown_timer = 10 SECONDS
```

Comment:
i just dont like much these, its a big movimentation skill for a tanky beno, don' think it needs its CD reduced to less than 15 seconds

Reply:
how about 12? that puts it inline with shrike single target fling

Reply:
seems better then, the plasma cost can be big as it was, crusher doenst use that much plasma in charge

Reply:
Okay making it 12

>rusher doenst use that much plasma in charge

The cost is reduced to match the cooldown so your plasma drains at the same speed

>(infact you can charge basically infinitely on weeds)

Incorrect, please test it yourself in long lanes if you don't believe me.